admin/data/sql/migrations/077.do.sql-session-store.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE [mtc_admin].[sessions](
+  [sid] varchar(255) NOT NULL
```

Comment:
If this is 255 chars and un-indexed won't retrievals be very very slow?

Reply:
its the primary key

Reply:
its the primary key